vllm_ascend/worker/model_runner_v1.py

```diff
@@ -706,6 +710,31 @@ def get_eagle_atten_dict(
     def get_model(self) -> nn.Module:
         return self.model
 
+    def get_supported_generation_tasks(self) -> "list[GenerationTask]":
```

Comment:
we should inherit from base runner in vLLM in the future instead of copy the code again and again. I'll do the refactor later